### PR TITLE
Tim/stable json

### PR DESCRIPTION
**Pull request by tux3**

### CODE REVIEW

File: tankersdk_identity/__init__.py [hunk 1]
```diff
@@ -52,6 +52,23 @@ def _deserialize_identity(identity: str) -> Dict[str, str]:
     return json.loads(identity_json)  # type: ignore
 
 
+def _to_ordered_json(obj: Dict[str, Any]) -> str:
+    keys = sorted(obj.keys())
+    items = []
+    for k in keys:
+        if type(obj[k]) == "dict":
```
[the remark this answers is not in the document]
Comment (hackedteam) [reply tankersdk_identity/__init__.py:59]: Isn't this more complicated than I need (https://stackoverflow.com/questions/42378726/why-is-checking-isinstancesomething-mapping-so-slow)?

[the remark this answers is not in the document]
Comment (tux3) [reply tankersdk_identity/__init__.py:59]: Isn't this more complicated than I need (https://stackoverflow.com/questions/42378726/why-is-checking-isinstancesomething-mapping-so-slow)?

File: tankersdk_identity/test/test_vectors.py [hunk 2]
```diff
@@ -87,3 +87,11 @@ def test_parse_valid_public_provisional_identity() -> None:
         identity["public_encryption_key"]
         == "/2j4dI3r8PlvCN3uW4HhA5wBtMKOcACd38K6N0q+mFU="
     )
+
+
+def test_ordered_json() -> None:
+    obj = {"c": 2, "b": None, "a": {"x": "y"}}
+    expected_json = '{"a":{"x":"y"},"b":null,"c":2}'
```
[the remark this answers is not in the document]
Comment (tux3) [reply tankersdk_identity/test/test_vectors.py:94]: Yes, agree. They should be in other languages, possibly with minor differences (e.g. Java only supports String->String)

File: tankersdk_identity/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (blastrock) [reply tankersdk_identity/__init__.py:59]: Hmm dunno, maybe some middle ground like `isinstance(obj, dict)` would be ok? I don't feel strongly about this, just trying to do as the roman in rome, you can discard this comment if you feel we don't care ^^

[the remark this answers is not in the document]
Comment (tux3) [reply tankersdk_identity/__init__.py:59]: No I'll use Mapping I suppose, after reading a bit more I guess it doesn't hurt to be compatible with the other (sub)classes. Thanks!

**Pull request closed without merging**